### PR TITLE
[UI] fixed dark mode issues in Meshery UI 

### LINKED ISSUE #7298: [UI] Header settings when creating new design should be dark mode compatible
#### Current Behavior
As you can see two issues in the below video
- settings header inside application section should contain `#00B39F` as background in dark mode
- when switching between dark mode and light mode text `settings` is displaced from its place

https://user-images.githubusercontent.com/71271069/223419340-d8877939-e827-4bad-bad4-769799038ffe.mov

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #7298 
check the below screenshot of the changes

Frame 3

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — COMMENTED** [review]
Thank you, @Yashsharma1911. If the "Settings" title bar isn't clickable, then let's not use the Keppel green (action color) for this title bar, but a different gray from the dark mode color palette instead.

**Comment by Yashsharma1911** [comment]
Screenshot 2023-04-01 at 9 57 22 PM

**Comment by Chadha93** [comment]
@Yashsharma1911  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

File: ui/components/MesheryApplications/ApplicationsCard.js [hunk 1]
```diff
@@ -49,6 +50,7 @@ function MesheryApplicationCard({
   };
   const theme = useTheme();
   const classes = useStyles()
+  const theme = useTheme()
```
Comment (Chadha93) [inline ui/components/MesheryApplications/ApplicationsCard.js:53]:
```suggestion
```

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**